Orange/canvas/report/owreport.py

```diff
@@ -337,14 +340,14 @@ def open_report(self):
 
         try:
             report = self.load(filename)
-        except (IOError, AttributeError) as e:
+        except (IOError, AttributeError):
             message_critical(
-                 self.tr("Could not load an Orange Report file"),
-                 title=self.tr("Error"),
-                 informative_text=self.tr("An unexpected error occurred "
-                                          "while loading '%s'.") % filename,
-                 exc_info=True,
-                 parent=self)
+                self.tr("Could not load an Orange Report file"),
+                title=self.tr("Error"),
+                informative_text=self.tr("An unexpected error occurred "
```

Comment:
Remove *unexpected* (all errors are unexpected) and probably also the article, hence "Error occurred while loading '%s'.